pages/facts.html

```diff
@@ -1,12 +1,58 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Fun Facts About Guy Fieri</title> 
+  <link rel="stylesheet" href="../styles/style.css"> 
+  <link rel="stylesheet" href="../styles/facts.css">
 </head>
-<body>
-  
+<body id="fact-body">
+  <header>
+    <h1 id="facts-h1">Fieri's Fact Page</h1>
+    <em>Too Factual too Fierious</em>
```

Comment:
The `em` tag is meant to be used when there is emphasis or stress on text that is important to the meaning https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em#usage_notes. In this case I would suggest using a `p` or `h2` element and style it as necessary with CSS.